blueprints/app/files/app/index.html

```diff
@@ -15,6 +15,10 @@
     {{content-for "head-footer"}}
   </head>
   <body>
+    <noscript>
+      JavaScript must be enabled to run this application.
```

Comment:
Well I am talking about new apps that are created will have fastboot addon added by default. Regarding addons, most addons should be compatible and if they are not it should be straightforward to make them compatible. 

In general, I am -1 to adding this `noscript` by default in the blueprint for the primary purpose that it is going to be confusing for the community.